third_party/xla_client/BUILD

```diff
@@ -47,8 +47,10 @@ tf_cc_shared_object(
             "-z defs",
             "-Wl,--version-script",  #  This line must be directly followed by the version_script.lds file
             "$(location //tensorflow/compiler/xla/xla_client:tf_version_script.lds)",
+            "-g",
```

Comment:
Are they still relevant in this PR? This was for debug builds?

Reply:
Right, this was for debugging. I've removed it from the later revisions.